sources/engine/Xenko.Graphics/Vulkan/CommandList.Vulkan.cs

```diff
@@ -42,6 +43,7 @@ public static CommandList New(GraphicsDevice device)
         private CommandList(GraphicsDevice device) : base(device)
         {
             Recreate();
+            dummyTexture = Texture.New2D(GraphicsDevice, 1, 1, PixelFormat.R8G8B8A8_UNorm_SRgb, TextureFlags.ShaderResource);
```

Comment:
Shouldn't this be part of `GraphicsDevice.Vulkan.cs` instead?

Reply:
Also, need to add a line to destroy it when `GraphicsDevice` is destroyed.

Reply:
Yes, it should. Working on another issue with Vulkan (https://github.com/phr00t/xenko/issues/49), so I'm not able to get to this right at the moment.

Reply:
This should be done.